deploy/fio.yaml

```
@@ -9,8 +9,7 @@ spec:
        claimName: ms-volume-claim
   containers:
     - name: fio
-      image: nixery.dev/shell/fio/tini
-      command: [ "tini", "--" ]
+      image: mayadata/fio
       args:
         - sleep
```

Shouldn't the `tini` stay somewhere? Without it it would take ages to shut down the container, wouldn't it?

Either `command` or in `args`?

```suggestion
        - tini
        - "--"
        - sleep
```

---

My understanding was that the command is defined in the Dockerfile. If not we definitely need to define it here and I suggest we use tail -f /dev/null (or similar) instead of sleep with very high value.

---

> My understanding was that the command is defined in the Dockerfile
It is not. But can be if someone puts it there :-)

---

I've added `ENTRYPOINT ["/sbin/tini", "--"]` into the container as it makes sense to have it baked into the container and I cannot imagine a use-case where one doesn't want to have it.

When run as: `docker run mayadata/fio sleep 8000` or `docker run mayadata/fio tail -f /dev/null` it works and when I try to stop the container (which without `tini` waits for a timeout) it works and is quick.

note: I've removed suggestion from above comment as it doesn't make sense now,

---

nice